rules/windows/process_creation/sysmon_long_powershell_commandline.yml

```diff
@@ -0,0 +1,26 @@
+title: Too Long PowerShell Commandlines
+id: 3f07b9d1-2082-4c56-9277-613a621983cc
+description: Detects Too long PowerShell command lines
+references:
+    - https://speakerdeck.com/heirhabarov/hunting-for-powershell-abuse
+tags:
+    - attack.execution
+    - attack.t1059.001
+status: experimental
+author: oscd.community, Natalia Shornikova
+date: 2020/10/06
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    Powershell_selection:
+      - CommandLine: 
+        - '*powershell*'
+        - '*pwsh*'
+      - Description: 'Windows Powershell'
+      - Product: 'PowerShell Core 6'
```

Comment:
the `Powershell_selection` (due to YAML syntax) is going to work this way:

```
CommandLine (something) OR Description (something) OR Product
```

Is it correct? I mean, is that what you've tried to achieve?

Reply:
> the `Powershell_selection` (due to YAML syntax) is going to work this way:
> 
> ```
> CommandLine (something) OR Description (something) OR Product
> ```
> 
> Is it correct? I mean, is that what you've tried to achieve?

Yes, that's what I need